kumo-core/src/main/java/com/kennycason/kumo/nlp/FrequencyAnalyzer.java

```diff
@@ -104,12 +104,18 @@ public FrequencyAnalyzer() {
     public List<WordFrequency> load(final InputStream inputStream, boolean autoFill) throws IOException {
         return load(IOUtils.readLines(inputStream, characterEncoding), autoFill, "nothing");
     }
+    //if enable autoFill but not specify autoFillWord, use nothing as default
+    //also allow use List<String> directly
     public List<WordFrequency> load(final List<String> texts, boolean autoFill){
         return load(texts, autoFill, "nothing");
     }
-
+    //If autoFill is false, just call load
+    //Else if there are some words to draw, repeat them until the number of characters is similar with wordFrequenciesToReturn
+    //Else repeat autoFillWord to satisfy the drawing cloud need.
+    //Note: the repeat time is dependence on the sum of word length. We consider character number is more important.
+    //Which means long word will be repeated less
     public List<WordFrequency> load(final List<String> texts, boolean autoFill, String autoFillWord) {
-        if(!autoFill){
+        if(autoFill == false){
```

Comment:
it's ok for now, but for future `if (!autoFill) {` was 100% ok with me. :)

Reply:
got it